contrib/rosetta/configuration/bootstrap.json

```diff
@@ -1,12 +1,12 @@
 [
   {
     "account_identifier": {
-      "address":"cosmos1hdmjfmqmf8ck4pv4evu0s3up0ucm0yjjqfl87e"
+      "address":"cosmos158nkd0l9tyemv2crp579rmj8dg37qty8lzff88"
```

Comment:
Ah, I understand. I updated the store keys, so the `data.tar.gz` should be updated (but the addresses kept same!)

How would I go and update the data.tar.gz file?

Reply:
ping @fdymylja, is there's a README/tutorial on how to generate this tar.gz?

Reply:
@amaurymartiny sorry for the late response! it slipped my mind.

There's a command in the make file: make rosetta-data.

It's gonna output 2 addresses, 1 AccAddress that needs to be substituted in bootstrap.json, and 1ValOper that needs to be substituted in staking.ros.

I didn't document this in the readme! my bad!

Reply:
I don't think I'm gonna document it because we are planning to change it soon. I will do it then.

Reply:
Got it, thanks!

Can i merge this PR then? I had a doubt about the val address change (it shouldn't change because of #8363), but it seems that the `make rosetta-data` outputted new addresses so it's okay.

Reply:
Well it outputted new addresses now that I rebuilt it. In your branch (before this PR merge) the address was unchanged (data.tar.gz wasn't changed in your branch). 

Which means that the address computed by distribution is now different from the one it computed before, so it was not found in the 'static' state contained in data.tar.gz.

It means that most likely, if we take the work on #8363, build a binary and try to restart a chain it's gonna fail.